curriculum/challenges/english/25-front-end-development/lecture-working-with-pseudo-classes-and-pseudo-elements-in-css/672bbed37c6f51aa3a15e78c.md

```diff
@@ -10,6 +10,159 @@ dashedName: what-are-pseudo-elements
 
 Watch the lecture video and answer the questions below.
 
+# --transcript--
+
+What are pseudo-elements, and how do they work?
+
+One of the most interesting aspects of CSS is the use of pseudo-elements. In this context, “pseudo” means “not real”, so pseudo-elements are virtual or synthetic elements that don't directly match any actual HTML elements. They allow you to style specific parts of an element or insert content without adding extra HTML.
+
+To apply a pseudo-element, attach it to the original element's selector using a double colon `::`. Note that the selector can be any type, such as a class or ID selector.
+
+Here's what the basic syntax of pseudo-elements looks like:
+
+```css
+selector::pseudo-element {
+ property: value;
+}
+```
+
+This double colon is what distinguishes pseudo-elements from pseudo-classes, which use a single colon.
+
+Pseudo-elements allow you to style specific parts of an element's content or insert content before or after it, but they cannot exist independently. The element to which a pseudo-element is attached is called its originating element.
+
+Let's start by looking at examples for the `::before` and `::after` pseudo-elements. As their names suggest, `::before` lets you insert content just before the element's content while `::after` lets you insert content after it.
+
+Here is an example of a button element:
+
+```html
+<button class="cta-button">Learn More</button>
+```
+
+In the CSS, we will use absolute positioning and the `::before` pseudo-element to add a star before the button's “Learn More” text. You will learn more about absolute positioning in future videos.
+
+```css
+.cta-button {
+  background-color: lightseagreen;
+  color: white;
+  border: none;
+  padding: 10px 20px;
+  cursor: pointer;
+  position: relative;
+}
+
+.cta-button::before {
+  content: "⭐";
+  position: absolute;
+  left: 3px;
+  top: 8px;
+  font-size: 0.75rem;
+}
+```
+
+The `content` property is used to represent the content you wish to add before the button text. In this example, we are adding a star. You'll notice that you can not only insert content but also style it.
+
+Here's an example of the `::after` pseudo-element with the same button:
+
+```css
+.cta-button {
+  background-color: orange;
+  border: none;
+  padding: 10px 30px;
+  cursor: pointer;
+  position: relative;
+}
+
+.cta-button::after {
+  content: '➡️';
+  position: absolute;
+  right: 5px;
+  bottom: 6px;
+  font-size: 1.125rem;
+  transition: transform 0.3s ease;
+}
+```
+
+In the `::after` pseudo-element, the `transition` property is used to animate changes over 0.3 seconds with an easing effect, creating a smooth and gradual transformation rather than a sudden one. You will learn more about the transition property in future videos. 
```

Comment:
```suggestion
In the `::after` pseudo-element, the `transition` property is used to animate changes over `0.3` seconds with an easing effect, creating a smooth and gradual transformation rather than a sudden one. You will learn more about the `transition` property in future videos. 
```